### ISSUE TITLE
Your app includes non compliant SDK version | FlurrySDK

### ISSUE THREAD

**Issue by teamelliel** [flurry/flurry-android-sdk#33]
Your app com.xxxxxx version code 247791927 includes SDK **com.flurry.android:analytics, 12.13.0 or an SDK** that one of your libraries depends on, which collects personal or sensitive data that includes but may not be limited to Android ID, Advertising ID identifiers. Persistent device identifiers may not be linked to other personal and sensitive user data or resettable device identifiers as described in the [User Data](https://support.google.com/googleplay/android-developer/answer/10144311) policy.

Starting from January 23, 2023 midnight (UTC), new app releases containing the SDK version(s) that do not comply with the [User Data](https://support.google.com/googleplay/android-developer/answer/10144311) policy may be blocked from release. You may consider upgrading to a policy-compliant version of this SDK that does not include the violating code if available from your SDK provider, or removing this SDK from your app.

According to your SDK provider, you may consider upgrading to **14.0.0**, and/or contacting your SDK provider to see if a suitable later version might be available. Google is unable to endorse or recommend any third party software.

**ACTION REQUIRED:** Upload a new compliant version AND deactivate the noncompliant version.

Read through the [User Data](https://support.google.com/googleplay/android-developer/answer/10144311) policy for more details, and how to submit an updated app for review [here](https://support.google.com/googleplay/android-developer/answer/2477981#resubmit).

If you've reviewed the policy and feel our decision may have been in error, please reach out to our [policy support team](https://support.google.com/googleplay/android-developer/contact/appwarning?ts=sdk&src=com.flurry.android:analytics).

**Comment by poting-oath** [comment]
As it says, you need to upgrade to Flurry SDK 14.0.0. From our release notes:

In Version 14.0.0 - 10/20/2022
https://developer.yahoo.com/flurry/docs/releasenotes/android/,

Report device identifiers and location collection to be Google Policy Compliant

Note: Flurry SDK 14.0.0 will not collect Android device ID when either Google Play Advertising ID or Location Report is collected.

**Comment by monospare** [comment]
my client got app rejection from google for version 14.0.0 also , please help me how to fix this?

Your app is using the Flurry SDK which collects but may not be limited to Android ID and Advertising ID identifiers. Persistent device identifiers may not be linked to other personal and sensitive user data or resettable device identifiers.

You may consider moving to another SDK; removing this SDK; or if available from your SDK provider,upgrading to a policy-compliant version of this SDK that does not include the violating code.

According to the information provided by your SDK provider, you may consider upgrading to 14.0.0. Please consult the SDK provider for further information.

**Comment by poting-oath** [comment]
@TheTanmay1 As it says, you need to upgrade to Flurry SDK 14.0.0.

**Comment by monospare** [comment]
yes i know and i was using 14.1.0 and then i also tried 14.0.0 still same error

**Comment by poting-oath** [comment]
> According to the information provided by your SDK provider, you may consider upgrading to 14.0.0. Please consult the SDK provider for further information.

@TheTanmay1 Please make sure your apps published do use Flurry SDK 14.0.0 or later to build.

**Comment by thenameisvijay** [comment]
@poting-oath  we are using Flurry SDK 14.0.0 in our app but still get rejected by Google. Got any solution for this? As there is an urgent release. Thanks.

**Comment by poting-oath** [comment]
@teamelliel @TheTanmay1 @thenameisvijay Please provide your app's package name. We will work with Google about this issue. Thanks! And please also double confirm the Flurry SDK version you are using.

**Comment by teamelliel** [comment]
The issue is even after updating the package to 14.0.1 or up, it still shows the same noncompliant SDK error.

**Comment by poting-oath** [comment]
@teamelliel There are hundreds of apps use 14.0.0 are already published on the Google Store. Please provide your app's package name. We will work with Google about this issue. Thanks!

**Comment by poting-oath** [comment]
@thenameisvijay Make sure your projects do use Flurry SDK 14.#.# or above. To see the full dependencies, please try to run `"gradlew dependencies"` under your project's app folder. Example of output,
```
...
+--- com.flurry.android:analytics:14.2.0
+--- com.flurry.android:marketing:14.2.0
|    \--- com.flurry.android:analytics:14.2.0
...
```
